### DISCUSSION TITLE
"Failed to program ESP"

### DISCUSSION THREAD

**Discussion by blasofsky** [forkineye/ESPixelStick#539] [General]
I am trying to flash to the esp via Mac. I get the Flash Tool open and connected but then get an error message of "failed to program ESP". I attached screen shots from the flash tool and my terminal. Thoughts
<img width="568" alt="Screen Shot 2022-06-02 at 10 40 46 AM" src="https://user-images.githubusercontent.com/106769015/171694751-ef0f5bf9-551c-468f-ba3c-b307d324a0ab.png">
<img width="568" alt="Screen Shot 2022-06-02 at 10 40 59 AM" src="https://user-images.githubusercontent.com/106769015/171694764-37d5caf4-a296-4304-b8eb-ec91718a08c2.png">
?

**Comment by MartinMueller2003** [comment]
you dont have the latest version of python installed on your mac

**Comment by blasofsky** [comment]
Fixed: I had the latest version installed. What I had to do was create an alias via Terminal of Python/Python3 because when its installed in Mac its installed as Python3. The flash program looks for Python, not Python3.
